### PR TITLE
Fixes #15161 Text area overflows its parent

### LINKED ISSUE #15161: [UI] Text area overflows its parent element
### Describe the bug

Text Area is overflowing its parent div

### To Reproduce

1. Select a DB
2. Click on 'Query'
3. Scroll down
4. See text area overflowing

### Expected behavior

It should not overflow its parent

### Screenshots

image

### Server configuration

- Operating system: Windows
- Web server: XAMPP
- PHP version: 7.3.0
- phpMyAdmin version: 5.0.0-dev

### Client configuration

- Browser: Firefox
- Operating system: Windows

### PR DESCRIPTION
**Pull request by AghaSaad04**
Signed-off-by: Agha Saad Fraz <agha.saad04@gmail.com>

### Description

I have added width for textarea and have removed margin as it was causing some layout issues.

Fixes #15161 

Before submitting pull request, please review the following checklist:

- [ ] Make sure you have read our [CONTRIBUTING.md](https://github.com/phpmyadmin/phpmyadmin/blob/master/CONTRIBUTING.md) document.
- [ ] Make sure you are making a pull request against the correct branch. For example, for bug fixes in a released version use the corresponding QA branch and for new features use the _master_ branch. If you have a doubt, you can ask as a comment in the bug report or on the mailing list.
- [ ] Every commit has proper `Signed-off-by` line as described in our [DCO](https://github.com/phpmyadmin/phpmyadmin/blob/master/DCO). This ensures that the work you're submitting is your own creation.
- [ ] Every commit has a descriptive commit message.
- [ ] Every commit is needed on its own, if you have just minor fixes to previous commits, you can squash them.
- [ ] Any new functionality is covered by tests.

### CODE REVIEW

**Comment by AghaSaad04** [comment]
Yes Sir @williamdes, I have just checked it only occurs on pmahomme.

File: themes/pmahomme/scss/_common.scss [hunk 1]
```diff
@@ -331,7 +331,7 @@ input {
 
 textarea {
   overflow: visible;
-  margin: 6px;
+  width: 100%;
```
Comment (mauriciofauth) [inline themes/pmahomme/scss/_common.scss]: Could you change the CSS of the specific textarea only instead of changing all textareas?

**Comment by mauriciofauth** [comment]
> So should I create PR for the rest themes?

@AghaSaad04 You can add commits to this pull request.

File: themes/pmahomme/scss/_common.scss (hunk 1 above)
Comment (AghaSaad04) [reply themes/pmahomme/scss/_common.scss]: Oh yeah, Sir, thanks for pointing out the issue, I'll be fixing it soon :)

**Pull request closed without merging**